src/bookshelf/src/bksw_conf.erl

```diff
@@ -129,6 +131,22 @@ port() ->
             {port, Port}
     end.
 
+rsync_uri() ->
+    case application:get_env(bookshelf, rsync_uri) of
+        undefined ->
+            undefined;
```

Comment:
Nitpick, should you just catch on
```
WhateverElse ->
   WhateverElse
```

Is that better erlang-ism? Just pass it along? Or do we wanna fail if undef (I'm actually asking lol).